ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -40,6 +40,14 @@ object Tests {
       new TransactionScaleIT,
     )
 
+  /**
+    * These are test suites that manipulate the ledger in such a way as they would perturb the results of other tests
+    */
+  val separate: Vector[LedgerTestSuite] =
```

Comment:
Thanks for the pointer. At some point I had been concerned that I would break other test suites particularly ones that read from the beginning of the ledger. I only found two such test (`TransactionServiceIT` "TXBeginToBegin" and "TXTreesBeginToBegin") cases after looking, but they only use empty ranges which do not result in pruning-related errors, so using this flag should work.

As a second layer of defense (that I don't think we want to rely on), the sequential tests are run after the concurrent tests, so if they leave a participant in such a state that concurrent tests no longer pass, that would be hidden from us. For now, I am adding a somewhat artificial test `ParticipantPruningIT."PRPruneQueryEmptyRangeOk"` that notifies us if querying empty ranges in pruned offsets start to error.